src/rules/fileNameCasingRule.ts

```diff
@@ -79,6 +93,15 @@ export class Rule extends Lint.Rules.AbstractRule {
         }
     }
 
+    private static shouldFileBeConsidered(
+        filesToIgnore: string[] | undefined,
```

Comment:
Since you're allowing `undefined` here, maybe you shouldn't just type-cast `ruleArguments[1]` to `string` (line 124). Also because it actually will be undefined in most of the cases :-)

Reply:
I've updated the type assertion on line 124 to match this parameter declaration.  Since `ruleArguments` is of type `any[]`, I should probably update that assertion to provide some form of narrowing of they types :-) Thanks for the suggestion!